scripts/test/test_launch_controller.py

```diff
@@ -67,10 +63,18 @@ def main():
         if args.cluster:
             extra_args += ["--cluster", "true", "--dumpAgencyOnError", "true"]
         extra_args += ["--testBuckets", args.testBuckets]
-        runner = TestingRunner(SiteConfig(Path(args.definitions).resolve()))
-        runner.scenarios.append(
-            TestConfig(runner.cfg, name, suite, [*extra_args], 1, 1, [])
-        )
+        config = SiteConfig(Path(args.definitions).resolve())
+        config.serialize_execution = True
+        runner = TestingRunner(config)
+        if args.allProtocols:
+            for proto in ["http", "http2", "vst"]:
```

Comment:
VST protocol is not supported anymore.
```suggestion
            for proto in ["http", "http2"]:
```